demo/list-box-styling-demos.html

```diff
@@ -0,0 +1,66 @@
+<dom-module id="list-box-styling-demos">
+    <template>
+      <style include="vaadin-component-demo-shared-styles">
+        :host {
+          display: block;
+        }
+      </style>
+
+    <h3>Custom input field style</h3>
```

Comment:
There is no "input field" in `vaadin-list-box` so the title is confusing.

Please rename the demo to something else, like "Customizing the appearance"

Reply:
Done.